### DISCUSSION TITLE
[Feature] Slideshow improvements

### DISCUSSION THREAD

**Discussion by jvgoch** [immich-app/immich#7902] [Feature Request]
### The feature

I would like to use immich to use an old tablet as a photo frame but currently there are a few things preventing that:

- The mobile app doesn't support slideshows at all, it would be great if this could be added. (More of a +1 since I'm not the first to ask for this.)

Using a webpage would work as well but there are some distracting ui issues.
- The slideshow on web doesn't have an option to hide the immich ui. (play/pause button etc.) This could be an auto hide feature if you don't use your mouse or don't touch the screen for a few seconds. Autohide could also be toggled in the new slideshow settings pop-up.
- The slideshow on web shows the ui of the browsers internal video player at the start of every video. (don't know if this can be fixed)

It would be great if these things could be added to create a seamless, non distracting slideshow experience on any device.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4084

**Comment by Pixette** [comment]
If you're after a native app rather than the web slideshow, full disclosure I'm the dev of Pixette (iOS/Android). It connects straight to your Immich server and runs a full-screen photo and video slideshow with no UI on screen, configurable interval, and shuffle, basically the seamless photo-frame experience described here. I built it to run on old hardware too (down to iOS 13 / an iPad Mini 2), since that's exactly the "revive an old tablet" use case. Not affiliated with Immich, just sharing in case it helps.
https://pixette.app

**Reply by bo0tzz** [reply]
@Pixette advertising a closed-source app with paywalls on an open source project's pages is not a great look.
